Makefile

```diff
@@ -429,7 +429,7 @@ $(BUILD_DIR)/src/code/__osMalloc_n64.o: OPTFLAGS := -O2
 $(BUILD_DIR)/src/code/__osMalloc_gc.o: OPTFLAGS := -O2
 $(BUILD_DIR)/src/code/code_800FC620.o: OPTFLAGS := -O2
 $(BUILD_DIR)/src/code/fp_math.o: OPTFLAGS := -O2
-$(BUILD_DIR)/src/code/rand.o: OPTFLAGS := -O2
+$(BUILD_DIR)/src/libc64/qrand.o: OPTFLAGS := -O2
```

Comment:
alright, I guess we're doing libc64 now. In that case I think this should be
```
$(BUILD_DIR)/src/libc64/%.o: OPTFLAGS := -O2
```
since this is a major motivation for moving the libc64 files.

Also I think this should be in a separate block, maybe closer to where `src/libc` is handled below. And should we rename src/libc to src/nustd to prevent confusion with libc64?

Reply:
moved libc64 source to src/libc64